src/index.html

```diff
@@ -9,8 +9,10 @@
   <link rel="preconnect" href="https://fonts.gstatic.com">
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
```

Comment:
Please load fonts and css from our project not cdns. I think they can be installed via npm

Reply:
I tried to install bootstrap via "npm i bootstrap" / "npm install bootstrap@3", but it does not work... maybe we can do it later and merge Branches now, so that Manuel will be able to work with a PR